cli/functionary/utils.py

```diff
@@ -45,6 +47,21 @@ def flatten(results, object_fields):
     return new_results
 
 
+def _fix_datetime_display(value):
+    """
+    Helper function for format_results to remove milliseconds from datetime
+
+    Args:
+        value: string representing datetime value
+    Returns:
+        value as a string representing datetime value without milliseconds
+    """
+    value = datetime.datetime.strptime(value, "%Y-%m-%dT%H:%M:%S.%fZ").replace(
```

Comment:
It's not obvious from the documentation, but I think using the %z format code will properly handle the `Z` designating UTC.  So this could just be:

```python
value = datetime.datetime.strptime(value, "%Y-%m-%dT%H:%M:%S.%f%z")
```